### PR TITLE
prep for move to Esri org

### PR DESCRIPTION
**Pull request by jgravois**
* Apache 2.0 license
* more `package.json`
* Esri copyright header in project entrypoint
* more README
* newer target node version

### CODE REVIEW

**Comment by jgravois** [comment]
i'll put in another inquiry with the admins to review and move this to @Esri as soon as this PR lands.

**Comment by jgravois** [comment]
i've rebased this PR.

@tomwayson if you can review/merge afterward we can move to https://github.com/ArcGIS afterward (and then https://github.com/Esri)

File: README.md [hunk 1]
```diff
@@ -251,16 +252,10 @@ The `/etc/hosts` equivalent filepath on Windows is:
...
-* `git clone` this repository
-* `npm install`
-* `bower install`
-
-## Running Test App
+## Running
 
-* `ember server`
-* Visit your app at http://localhost:4200.
+* `ember server --dummy`
```
Comment (tomwayson) [inline README.md]: is `--dummy really needed`?

Comment (jgravois) [reply README.md]: no. neither is the **r** in _server_ as far as i can tell.

**Comment by jgravois** [comment]
pushed another commit to address @tomwayson's feedback.

i don't have push access to this repo to merge this PR, let alone admin access to stage it temporarily in the ArcGIS Organization.

**Comment by tomwayson** [comment]
@dbouwman will have to transfer to the ArcGIS org

**Pull request merged**